openedx/core/lib/request_utils.py

```diff
@@ -125,26 +125,26 @@ def process_exception(self, request, exception):
         _log_and_monitor_expected_errors(request, exception, 'middleware')
 
 
-# .. setting_name: EXPECTED_ERRORS
+# .. setting_name: IGNORED_ERRORS
 # .. setting_default: None
 # .. setting_description: Used to configure logging and monitoring for expected errors.
 #     This setting is configured of a list of dicts. See setting and toggle annotations for
-#     ``EXPECTED_ERRORS[N]['XXX']`` for details of each item in the dict.
+#     ``IGNORED_ERRORS[N]['XXX']`` for details of each item in the dict.
 #     If this setting is a non-empty list, all uncaught errors processed will get a ``checked_error_expected_from``
 #     attribute, whether they are expected or not. Those errors that are processed and match a 'MODULE_AND_CLASS'
 #     (documented elsewhere), will get an ``error_expected`` custom attribute. Unexpected errors would be errors with
```

Comment:
Ultimately we'll want to search for all uses of the word "expected", including its use in the word "Unexpected".